consistenthash/consistenthash_test.go

```diff
@@ -34,6 +34,13 @@ func TestHashing(t *testing.T) {
 		return uint32(i)
 	})
 
+	// Get a value before any key is added to the hash
+	// Should yield an empty string
+	empty_value := hash.Get("empty key")
+	if empty_value != "" {
+		t.Errorf("Expecting empty string, got %s", empty_value)
```

Comment:
Go style for errors is:

    Func(input) = got; want foo

Also, use %q for strings when the string isn't what you expect. %q shows empty strings and strings with binary gibberish like newlines well.

I'd write this all as:

```go
    if got := hash.Get("foo"); got != "" {
          t.Errorf("Get on empty hash = %q; want empty string", got)
    }
```